### PR TITLE
include all xforms doc types in fast reindex

### PR DESCRIPTION
**Pull request by dannyroberts**
http://manage.dimagi.com/default.asp?230376

### CODE REVIEW

**Comment by dannyroberts** [comment]
@proteusvacuum @esoergel @sravfeyn

File: corehq/apps/hqcase/management/commands/ptop_fast_reindex_xforms.py [hunk 1]
```diff
@@ -17,11 +17,30 @@ class Command(ElasticReindexer):
...
+        # copied from couchforms/_design/filters/xforms.js
+        doc_types = [
+            'XFormInstance',
+            'XFormArchived',
+            'XFormError',
+            'XFormDeprecated',
+            'XFormDuplicate',
+            'XFormInstance-Deleted',
+            'HQSubmission',
+            'SubmissionErrorLog',
```
Comment (sravfeyn) [inline corehq/apps/hqcase/management/commands/ptop_fast_reindex_xforms.py]: @dannyroberts From quick glance at the ticket, it looks addionally `XFormDuplicate` docs are to be indexed. Curious why you added other doc_types (for e.g. `XFormInstance-Deleted`) to indexer.

Comment (proteusvacuum) [inline corehq/apps/hqcase/management/commands/ptop_fast_reindex_xforms.py]: do we also need to update https://github.com/dimagi/commcare-hq/blob/master/corehq/pillows/xform.py#L193 ?

**Comment by proteusvacuum** [comment]
@dannyroberts, merging - can respond in a follow up, if necessary.

File: corehq/apps/hqcase/management/commands/ptop_fast_reindex_xforms.py (hunk 1 above)
Comment (dannyroberts) [inline corehq/apps/hqcase/management/commands/ptop_fast_reindex_xforms.py]: @sravfeyn these are all the doc types that are pulled in by realtime pillowtop, so I'm making the fast reindexing do the same thing

Comment (dannyroberts) [inline corehq/apps/hqcase/management/commands/ptop_fast_reindex_xforms.py]: @proteusvacuum oh fascinating. Are we in the process of switching from one to the other? I think I just missed that; I don't recognize that function

Comment (proteusvacuum) [inline corehq/apps/hqcase/management/commands/ptop_fast_reindex_xforms.py]: AFAIK, yes. The `v2` reindexers do Kafka stuff.

Comment (dannyroberts) [inline corehq/apps/hqcase/management/commands/ptop_fast_reindex_xforms.py]: Ah, must be simon's recent work?

Comment (dannyroberts) [inline corehq/apps/hqcase/management/commands/ptop_fast_reindex_xforms.py]: Cool. Thanks!

**Pull request merged**